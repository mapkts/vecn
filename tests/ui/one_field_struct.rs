#[vecn::vector]
struct TupleStruct<T>(T);

#[vecn::vector]
struct NamedStruct<T> {
    x: T,
}

fn main() {}
