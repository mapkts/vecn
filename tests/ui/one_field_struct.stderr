error: create a vector with only one field is not allowed
 --> $DIR/one_field_struct.rs:2:22
  |
2 | struct TupleStruct<T>(T);
  |                      ^^^

error: create a vector with only one field is not allowed
 --> $DIR/one_field_struct.rs:5:23
  |
5 |   struct NamedStruct<T> {
  |  _______________________^
6 | |     x: T,
7 | | }
  | |_^
